pkgs/top-level/python-packages.nix

```diff
@@ -696,12 +696,16 @@ in {
 
   aiofiles = callPackage ../development/python-modules/aiofiles { };
 
+  aioh2 = callPackage ../development/python-modules/aioh2 { };
+
   aiohttp = callPackage ../development/python-modules/aiohttp { };
 
   aiohttp-cors = callPackage ../development/python-modules/aiohttp/cors.nix { };
 
   aiohttp-jinja2 = callPackage ../development/python-modules/aiohttp-jinja2 { };
 
+  aiohttp_remotes = callPackage ../development/python-modules/aiohttp_remotes { };
```

Comment:
Please use `-`

Reply:
I did initially, but the name on Pypi is with an underscore. The package only works if `pname` uses an underscore. Is it still right to use a dash for the attribute name even so?

Reply:
Yes it is: https://nixos.org/nixpkgs/manual/#contributing-guidelines